lib/cc/analyzer/issue_location_format_validation.rb

```diff
@@ -30,13 +30,13 @@ def valid_positions?(positions)
       def valid_position?(position)
         position &&
           (
-            (position["line"] && position["column"]) ||
-            position["offset"]
+            [position["line"], position["column"]].all? { |value| value.is_a?(Integer) } ||
+            position["offset"] && position["offset"].is_a?(Integer)
```

Comment:
Does this mean you're going to replace the errors we saw in production with engine validation errors?

If so, would it be better to cast?